doc/whats_new.rst

```diff
@@ -44,6 +44,10 @@ Enhancements
 Bug fixes
 .........
 
+   - :class:`sklearn.feature_selection.SelectFdr` now correctly works
```

Comment:
A more descriptive message to tell people what was broken and is now fixed would be helpful. Often we use a message like "Fixed a bug where :class:`feature_selection.SelectFdr` did not ..."